native/cocos/2d/renderer/UIModelProxy.cpp

```diff
@@ -87,7 +87,7 @@ void UIModelProxy::uploadData() {
     for (size_t i = 0; i < drawInfos.size(); i++) {
         auto* drawInfo = drawInfos[i];
         auto* ia = subModelList.at(i)->getInputAssembler();
-        if (drawInfo->getVertexOffset() <= 0) continue;
+        if (drawInfo->getVertexOffset() <= 0 || drawInfo->getIndexOffset() <= 0) continue;
```

Comment:
why need this?

Reply:
Because graphics fills the vb first when building the mesh, then earcut finds the mesh is not available when organizing the ib, resulting in vertexOffset > 0 && indexOffset = 0, so it needs to be protected here.
The reproducible code snippet is as follows：
```ts
        g.fillColor = Color.WHITE;
        g.lineWidth = 2.2;
        g.clear();
        g.moveTo(0, 0);        
        g.lineTo(0, 0);
        g.lineTo(0, 0);
        g.close();
        g.fill();
```